readme.rst

```diff
@@ -3,63 +3,42 @@ Documentation starter pack
 
 See the `Sphinx and Read the Docs <https://canonical-documentation-with-sphinx-and-readthedocscom.readthedocs-hosted.com/>`_ guide for instructions on how to get started with Sphinx documentation.
 
-Then go through the following sections to use this starter pack to set up your docs repository.
+Then go through the following sections to use this starter pack to set up your documentation repository.
 
-Set up your documentation repository
-------------------------------------
+Prerequisites
+-------------
 
-You can either create a standalone documentation project based on this repository or include the files from this repository in a dedicated documentation folder in an existing code repository.
+Before you begin, ensure you have the following:
 
-**Note:** We're planning to provide the contents of this repository as an installable package in the future, but currently, you need to copy and update the required files manually.
+* A GitHub repository where you want to host your documentation, cloned to your local machine. The recommended approach is to host the documentation alongside your code in a `docs` folder. But a standalone documentation repository is also an option; in this case, start with an empty repository.
+* Git and Bash installed on your system.
 
-Standalone documentation repository
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Initialise your documentation repository
+---------------------------------------
 
-To create a standalone documentation repository, clone this starter pack
-repository, `update the configuration <#configure-the-documentation>`_, and
-then commit all files to the documentation repository.
+The ``init.sh`` Bash script is used to initialize your repository with the starter pack content. It adds all the files to your repository that are needed to get started with Sphinx documentation.
 
-You don't need to move any files, and you don't need to do any special
-configuration on Read the Docs.
+To use the script, follow these steps:
 
-Here is one way to do this for newly-created fictional docs repository
-``canonical/alpha-docs``:
+* Copy ``init.sh`` to your repository's root directory.
```

Comment:
Shouldn't we have the script do this?

Reply:
It's a manual step either way. Copy to repo, run. Vs. run, enter repo path.

Reply:
If the script copies the file then the user does not need to.